fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/services/AbstractDeleteResourceService.java

```diff
@@ -126,6 +134,8 @@ private void deleteDepthFirst(final Transaction tx, final PersistentStorageSessi
             final FedoraResource acl = fedoraResource.getAcl();
             if (acl != null) {
                 doAction(tx, pSession, acl.getFedoraId(), userPrincipal);
+                // Flush ACL cache on any ACL creation/update/deletion.
+                authHandleCache.invalidateAll();
```

Comment:
Is it possible to delete the ACL directly? It seems like the acl cache would get flushed if you delete a resource with an acl, but not if you delete the acl by itself.

Reply:
Ohh good catch, yes let me add for that.